core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -279,7 +282,7 @@ public Table loadTable(SessionContext context, TableIdentifier identifier) {
     LoadTableResponse response;
     TableIdentifier loadedIdent;
     try {
-      response = loadInternal(context, identifier);
+      response = loadInternal(context, identifier, paths.snapshotLoading());
```

Comment:
I constantly keep thinking that we're passing an actual path here because of `paths`, while it's actually an enum. Would be great if we could make this more obvious that it's not a path. Maybe we can move the enum to a different place?

Reply:
This is now removed